scripts/build_example

```diff
@@ -79,16 +89,18 @@ case "${language}" in
     exit 1;;
 esac
 
-if [[ "${docker_image}" == true ]]; then
+if [[ -n "${docker_config}" ]]; then
   if [[ "${language}" != "rust" ]]; then
     echo "Docker images are only supported for Rust modules"
     exit 1
   fi
 
-  bazel build "${bazel_build_flags[@]}" "//examples/${EXAMPLE}/server:${EXAMPLE}_image.tar"
+  readonly DOCKER_BAZEL_BUILD_FLAGS=("${bazel_build_flags[@]}" "--config=${docker_config}")
```

Comment:
I think is more confusing than helpful, perhaps just do this below

```
bazel build "${bazel_build_flags[@]}" "--config=${docker_config}" "//examples/${EXAMPLE}/server:${EXAMPLE}_image.tar"
```

up to you really, anyways perhaps add a comment